test/test_heuristics.rb

```diff
@@ -633,6 +633,7 @@ def test_m_by_heuristics
       "M" => all_fixtures("M", "MDB.m"),
       "Mathematica" => all_fixtures("Mathematica", "*.m") - all_fixtures("Mathematica", "Problem12.m"),
       "MATLAB" => all_fixtures("MATLAB", "create_ieee_paper_plots.m"),
+      "Octave" => all_fixtures("Octave", "accumarray.m"),
```

Comment:
As there are quite a few samples, we want this heuristic to correctly identify all of them correctly, not just the one file.

```suggestion
      "Octave" => all_fixtures("Octave", "*.m"),
```

Reply:
The very large sample files have been removed. The remaining ones express Octave syntax in a suitable manner. Corrected the heuristic to correctly identify all available sample files of the Octave language.